cargo-shuttle/src/lib.rs

```diff
@@ -847,7 +847,12 @@ impl Shuttle {
         let table = get_deployments_table(&deployments, proj_name, page, raw, page_hint);
 
         println!("{table}");
-        println!("Run `cargo shuttle logs <id>` to get logs for a given deployment.");
+
+        if deployments.is_empty() {
+            println!("Run `cargo shuttle deploy` to deploy your project.");
+        } else {
+            println!("Run `cargo shuttle logs <id>` to get logs for a given deployment.");
+        }
```

Comment:
I think the table call should be skipped as well if empty

Reply:
@jonaro00 It returns this message: `No deployments are linked to this service` which is nice to have.